<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Ширина канала в месте установки решетки, мм</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Глубина канала в месте установки решетки, мм</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Прозор решетки, мм</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Степень защиты привода</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Высота выгрузки отбросов, мм</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Материал исполнения</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_gap">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_cw">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_ch">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_ip">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_mat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_hl">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>380</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calculate">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>380</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Комплектный шкаф управления</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_cp">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Позиция по проекту</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_position">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cmb_protokol">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>270</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Протокол связи</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lb_img">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>441</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>459</x>
      <y>10</y>
      <width>31</width>
      <height>351</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mb_file">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="menu_bth_renew"/>
   </widget>
   <widget class="QMenu" name="mb_pravka">
    <property name="title">
     <string>Правка</string>
    </property>
    <addaction name="mn_globalParams"/>
    <addaction name="mn_price_params"/>
   </widget>
   <addaction name="mb_file"/>
   <addaction name="mb_pravka"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mn_globalParams">
   <property name="text">
    <string>Параметры оборудования</string>
   </property>
  </action>
  <action name="menu_bth_renew">
   <property name="text">
    <string>Обновить ТКП</string>
   </property>
  </action>
  <action name="mn_price_params">
   <property name="text">
    <string>Редактор цен</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
